src/_sass/_secondary_nav.scss

```diff
@@ -0,0 +1,196 @@
+$background-nav-color: #e76c7a;
+$secondary-nav-color: white;
+$current-color: white;
+
+.site-nav {
+  background-color: $background-nav-color;
+  height: 4.5rem;
+  position: fixed;
+  top: 4.5rem;
+  // transition: top cubic-bezier(0.71,-0.04, 0.32, 1.07) .2s;
+  width: 100%;
+  z-index: 999;
+  
+
+  .logo {
+    margin-top: 0.5625rem;
+    // height: 2.5rem;
+  }
+
+  .content {
+    align-items: center;
+    background-color: $background-nav-color;
+    display: flex;
+    float: none;
```

Comment:
.content has a float property? this wasn't the case when i last checked and if it was added in that's a huge change since it's a shared class.. lmk